### PR TITLE
Publish Python library to PyPi.

### PR DESCRIPTION
**Pull request by zongweil**
This allows users to install the library using pip.

https://pypi.org/project/openlocationcode/

### CODE REVIEW

**Comment by zongweil** [comment]
#137

**Comment by zongweil** [comment]
Need to investigate why the test can't find the module anymore.

File: python/README.md [hunk 1]
```diff
@@ -48,3 +56,23 @@ Run the unit tests and benchmarks locally with:
 bazel test python:openlocationcode_test
 ```
 
+
+## Releasing to PyPi
```
Comment (tvews8z) [inline python/README.md:60]: https://github.com/google/open-location-code/releases/tag/1.0.0

**Review by tvews8z — APPROVED** [review]

File: python/README.md (hunk 1 above)
Comment (Tinar58) [reply python/README.md:60]: What is this?

**Review by jirasakmu — APPROVED** [review]
https://github.com/google/open-location-code/blob/master/python%2Fsetup.py

**Pull request merged**